src/trunk.c

```diff
@@ -8046,20 +8173,58 @@ trunk_print_memtable(trunk_handle *spl, platform_log_handle *log_handle)
    for (uint64 mt_gen = mt_gen_start; mt_gen != mt_gen_end; mt_gen--) {
       memtable *mt = trunk_get_memtable(spl, mt_gen);
       platform_log(log_handle,
-                   "%lu: gen %lu ref_count %u state %d\n",
+                   "Memtable root_addr=%lu: gen %lu ref_count %u state %d\n",
                    mt_gen,
                    mt->root_addr,
                    allocator_get_ref(spl->al, mt->root_addr),
                    mt->state);
+
+      memtable_print(log_handle, spl->cc, mt);
    }
    platform_log(log_handle, "\n");
 }
 
+/*
+ * trunk_print()
+ *
+ * Driver routine to print a SplinterDB trunk, and all its sub-pages.
+ * Caller can supply non-NULL user_stream handle to redirect output.
```

Comment:
This sentence doesn't make sense. Caller must supply a non-NULL handle. Moreover, what is a "user_stream handle"?

Reply:
Deleted. Was a left-over comment from prior version of this print interface. (Thanks for catching it!)